src/Item/AbstractIdentifiableItem.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Everlution\Navigation\Item;
+
+
+/**
+ * Class AbstractItem
+ *
+ * @author Martin Adamik <martin.adamik@everlution.sk>
+ */
+abstract class AbstractIdentifiableItem implements IdentifiableInterface
```

Comment:
this should be a trait without the constructor, an abstract class is too restrictive